vignettes_source/pkgnet-intro.Rmd

```diff
@@ -1,346 +0,0 @@
----
```

Comment:
whoa whoa whoa why is this disappearing? Was this a mistake that you left behind during testing?

Reply:
I'm still confused why this is disappearing...

Reply:
@bburns632 can you explain this